tests/v2/actions/clusters/verify.go

```diff
@@ -0,0 +1,39 @@
+package clusters
+
+import (
+	"errors"
+	"net/url"
+
+	steveV1 "github.com/rancher/shepherd/clients/rancher/v1"
+	"github.com/sirupsen/logrus"
+)
+
+const (
+	labelWorker = "labelSelector=node-role.kubernetes.io/worker=true"
+)
+
+var (
+	SmallerPoolClusterSize = errors.New("Machine pool cluster size is smaller than expected pool size")
+)
+
+// VerifyNodePoolSize is a helper function that checks if the machine pool cluster size is greater than or equal to poolSize
+func VerifyNodePoolSize(steveClient *steveV1.Client, poolSize int) error {
+	logrus.Info("Checking node pool")
+
+	logrus.Infof("Getting the node using the label [%v]", labelWorker)
```

Comment:
since this is hard coded to use the worker label, please update the function name to reflect that it exclusively will check for all nodes with a 'worker' label. note this is different than a node pool. i.e. I could have 2 node pools, one with 3 nodes, each having all roles. The other with 3 worker nodes. 

Since you promoted this to an action, it needs to be either more generic in what it can handle (maybe add a parameter for the labels you're looking for?) or more specific in the function name so that when someone else comes to use it, we know exactly what it is doing based on the name or godoc.